src/nvme/util.c

```diff
@@ -550,3 +559,154 @@ char *hostname2traddr(struct nvme_root *r, const char *traddr)
 	freeaddrinfo(host_info);
 	return ret_traddr;
 }
+
+char* startswith(const char *s, const char *prefix)
+{
+	size_t l = strlen(prefix);
+	if (strncmp(s, prefix, l) == 0)
```

Comment:
new line between deceleration and code.

btw, you are not even using the helper (which I don't disagree strongly to introduce it)